postgres/infoschema.go

```diff
@@ -336,6 +342,87 @@ func getConstraints(conv *internal.Conv, db *sql.DB, table schemaAndName) ([]str
 	return primaryKeys, m, nil
 }
 
+// getForeignKeys return list all the foreign keys constraints.
+func getForeignKeys(conv *internal.Conv, db *sql.DB, table schemaAndName) (foreignKeys []schema.ForeignKey, err error) {
```

Comment:
Similar comment to that in mysql/infoschema.go: Why do we need to getRefTables?
Couldn't we get the ref table from the select statement in getForeignKey.

Reply:
I have updated it as per the approach suggested by you. Please review it and then we can remove the commented previous approach.